sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -42,210 +38,221 @@ In the case of Service Bus, the modern client libraries have packages and namesp
 
 ### Client hierarchy
 
-In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`, rather than one for each of queue, topic and subscription:
+In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`, rather than one for each of queue, topic, subscription and session. This acts as the single entry point in contrast with multiple entry points from before. You can create senders and receivers from this client to the queue/topic/subscription/session of your choice and start sending/receiving messages.
 
-### Client constructors
+#### Approachability
+By having a single entry point, the `ServiceBusClient` helps with the discoverability of the API as you can explore all available features through methods from a single client, as opposed to searching through documentation or exploring namespace for the types that you can instantiate. Whether sending or receiving, using sessions or not, you will start your applications by constructing the same client.
+ 
+#### Consistency
+Having similarly named and structured methods to create senders, receivers and receivers for individual sessions on the same client provides consistency and predictability on the various features of the library. We have attempted to have the session/non-session usage be as seamless as possible. This allows you to make less changes to your code when you want to move from sessions to non-sessions or the other way around.
```

Comment:
> Having similarly named and structured methods to create senders, receivers and receivers for individual sessions on the same client provides consistency and predictability on the various features of the library. 

What does it mean, "individual sessions"? I guess the idea was to provide an opening for the second sentence right after, but it's confusing in the first sentence.

Reply:
By "receivers for individual sessions", I meant what you used to get from the Track 1 `AcceptMessageSessionAsync()` method. I have reworded the sentence. Is the below more clear?

```suggestion
We now have methods with similar names, signature and location to create senders, receivers and receivers for separate sessions. This provides consistency and predictability on the various features of the library. We have attempted to have the session/non-session usage be as seamless as possible. This allows you to make less changes to your code when you want to move from sessions to non-sessions or the other way around.
```

Reply:
Better. In the first sentence

> We now have methods with similar names, signature and location to create senders, receivers and receivers for separate sessions.

Is the intent to mention receivers for **specific** sessions?

Reply:
This is for the `createSessionReceiverAsync()` method that gives you a receiver for the session you asked for or if you havent specified a session, it gives you one for the next available one. 

I can skip the part on individual/separate sessions altogether. The following line talks about ease of moving from sessions to non sessions

```suggestion
We now have methods with similar names, signature and location to create senders and receivers. This provides consistency and predictability on the various features of the library. We have attempted to have the session/non-session usage be as seamless as possible. This allows you to make less changes to your code when you want to move from sessions to non-sessions or the other way around.
```

Reply:
Yeah. Omitting that details might be easier to digest the paragraph. 👍 for the [latest suggestion](https://github.com/Azure/azure-sdk-for-net/pull/14166#discussion_r486648489).